vumi/middleware/tests/test_message_storing.py

```diff
@@ -15,7 +15,8 @@ def setUp(self):
 
         # Create and stash a riak manager to clean up afterwards, because we
         # don't get access to the one inside the middleware.
-        self.persistence_helper.get_riak_manager()
+        manager = self.persistence_helper.get_riak_manager()
```

Comment:
I'm talking about the tests where we actually replace the code-under-test's Riak manager setup with our own -- `test_vumi_model_migrator.py`, for example.

Except in those places we're just handing out copies of our own manager instead of creating new ones, which means we're still not testing that those things properly close their connections. I'll fix that.